crates/driver/src/domain/competition/order/mod.rs

```diff
@@ -39,6 +40,9 @@ pub struct Order {
     pub sell_token_balance: SellTokenBalance,
     pub buy_token_balance: BuyTokenBalance,
     pub signature: Signature,
+    /// The types of fees that should be collected by the protocol.
+    /// The driver is expected to apply the fees in the order they are listed.
```

Comment:
Here we can document this driver's implementation (which is to apply the fee to the solutions it received)

```suggestion
  /// The types of fees the protocol collects from the winning solver.
  /// Unless otherwise configured, the driver modifies solutions to take sufficient fee in the form of positive slippage.
```

Reply:
added